collections.go

```diff
@@ -659,13 +663,13 @@ func checkUserForCollection(app *app, cr *collectionReq, r *http.Request, isPost
 	return u, nil
 }
 
-func newDisplayCollection(c *Collection, cr *collectionReq, page int) *DisplayCollection {
+func newDisplayCollection(c *Collection, cr *collectionReq, page int, reversed bool) *DisplayCollection {
```

Comment:
Again let's use `chrono` or maybe `forceChrono` as the parameter name, instead, to make this less confusing.